scripts/options/options.js

```diff
@@ -88,37 +123,7 @@ function restoreOptions() {
 	});
 }
 
-document.addEventListener('DOMContentLoaded', restoreOptions);
-
-const onClickElements = [
-	'animal-crossing',
-	'wild-world',
-	'new-leaf',
-	'game-random',
-	'sunny',
-	'snowing',
-	'raining',
-	'live',
-	'weather-random',
-	'no-kk',
-	'enable-kk',
-	'always-kk',
-	'enable-town-tune',
-	'enable-notifications',
-	'enable-badge',
-	'kk-version-live',
-	'kk-version-aircheck',
-	'kk-version-both'
-];
-
-document.getElementById('volume').oninput = saveOptions;
-
-onClickElements.forEach(el => {
-	document.getElementById(el).onclick = saveOptions;
-});
-
-let updateLocationEl = document.getElementById('update-location');
```

Comment:
Was this line removed by mistake?  I'm getting the following error when clicking the UpdateLocation button:
```
Uncaught ReferenceError: updateLocationEl is not defined
    at HTMLButtonElement.validateWeather (options.js:127)
```